### PR TITLE
chore: release 1.1.0

### PR DESCRIPTION
**Pull request by tolerants**
Release 1.1.0

### CODE REVIEW

File: libs/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,41 @@
...
+* **search:** facets and browser history ([#770](https://github.com/spryker/oryx/issues/770)) ([b3ba474](https://github.com/spryker/oryx/commit/b3ba474cd44e2df7909e09281010087bcca50a9a))
+* **search:** source for the query that is using for search ([#773](https://github.com/spryker/oryx/issues/773)) ([cd0ddbf](https://github.com/spryker/oryx/commit/cd0ddbfa3ce0dc282100f2c5e6d37adcf947d2f2))
+* **ui:** button states styles ([#771](https://github.com/spryker/oryx/issues/771)) ([7ffd7eb](https://github.com/spryker/oryx/commit/7ffd7ebb417ef59c87cc4e6ee9f4ed642501d96c))
+* **ui:** modal's height in safari ([#738](https://github.com/spryker/oryx/issues/738)) ([4360576](https://github.com/spryker/oryx/commit/4360576d9b24ed53c95b9e26883f4191446d43b3))
+* **ui:** modal's margins and scrolling behaviour  ([#736](https://github.com/spryker/oryx/issues/736)) ([506c84a](https://github.com/spryker/oryx/commit/506c84a41f07bcc56aca72adad9ce9c661e8acd6))
+
+
+### Features
```
Comment (tobi-or-not-tobi) [inline libs/CHANGELOG.md]: can we start with features first now or going forward?

File: libs/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,41 @@
...
+
+* **content:** resolve product, category and content links ([#775](https://github.com/spryker/oryx/issues/775)) ([f9f51e2](https://github.com/spryker/oryx/commit/f9f51e220cbf4a03ec2111d1789cc8be22b3ff3e))
+* **content:** update content service to possibility use adapters by config ([#755](https://github.com/spryker/oryx/issues/755)) ([43089dc](https://github.com/spryker/oryx/commit/43089dc3326448c58f99978e8c42385f238ccc2f))
+* introduce build time feature level flag, to accomodate breaking changes and feature versioning. ([#725](https://github.com/spryker/oryx/issues/725)) ([d7d6613](https://github.com/spryker/oryx/commit/d7d6613c39554a2e83768815082dbe3bca59a1e9))
+* **presets:** add b2b-storefront preset ([#703](https://github.com/spryker/oryx/issues/703)) ([b62f662](https://github.com/spryker/oryx/commit/b62f6625f5293ec8887a585f901794ca5514f3a9))
```
Comment (tobi-or-not-tobi) [inline libs/CHANGELOG.md]:
```suggestion
* **presets:** b2b-storefront preset ([#703](https://github.com/spryker/oryx/issues/703)) ([b62f662](https://github.com/spryker/oryx/commit/b62f6625f5293ec8887a585f901794ca5514f3a9))
```

File: libs/CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,41 @@
...
+* **ui:** modal's height in safari ([#738](https://github.com/spryker/oryx/issues/738)) ([4360576](https://github.com/spryker/oryx/commit/4360576d9b24ed53c95b9e26883f4191446d43b3))
+* **ui:** modal's margins and scrolling behaviour  ([#736](https://github.com/spryker/oryx/issues/736)) ([506c84a](https://github.com/spryker/oryx/commit/506c84a41f07bcc56aca72adad9ce9c661e8acd6))
+
+
+### Features
+
+* **content:** resolve product, category and content links ([#775](https://github.com/spryker/oryx/issues/775)) ([f9f51e2](https://github.com/spryker/oryx/commit/f9f51e220cbf4a03ec2111d1789cc8be22b3ff3e))
+* **content:** update content service to possibility use adapters by config ([#755](https://github.com/spryker/oryx/issues/755)) ([43089dc](https://github.com/spryker/oryx/commit/43089dc3326448c58f99978e8c42385f238ccc2f))
+* introduce build time feature level flag, to accomodate breaking changes and feature versioning. ([#725](https://github.com/spryker/oryx/issues/725)) ([d7d6613](https://github.com/spryker/oryx/commit/d7d6613c39554a2e83768815082dbe3bca59a1e9))
```
Comment (tobi-or-not-tobi) [inline libs/CHANGELOG.md]:
```suggestion
* feature level flag to opt-in using breaking changes during a major version lifecycle. ([#725](https://github.com/spryker/oryx/issues/725)) ([d7d6613](https://github.com/spryker/oryx/commit/d7d6613c39554a2e83768815082dbe3bca59a1e9))
```

File: libs/CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,41 @@
...
+* **presets:** add b2b-storefront preset ([#703](https://github.com/spryker/oryx/issues/703)) ([b62f662](https://github.com/spryker/oryx/commit/b62f6625f5293ec8887a585f901794ca5514f3a9))
+* **site:** breadcrumbs ([#735](https://github.com/spryker/oryx/issues/735)) ([8d9ea37](https://github.com/spryker/oryx/commit/8d9ea37fc1ee4decdd40b8a0357909017a2cfd5b))
+* **user:** sign up ([#785](https://github.com/spryker/oryx/issues/785)) ([968fb71](https://github.com/spryker/oryx/commit/968fb71212001efdb54737371c6c344936edaacf))
```
Comment (tobi-or-not-tobi) [inline libs/CHANGELOG.md]:
```suggestion
* **user:** user register page ([#785](https://github.com/spryker/oryx/issues/785)) ([968fb71](https://github.com/spryker/oryx/commit/968fb71212001efdb54737371c6c344936edaacf))
```

File: libs/CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,41 @@
...
+
+
+### Features
+
+* **content:** resolve product, category and content links ([#775](https://github.com/spryker/oryx/issues/775)) ([f9f51e2](https://github.com/spryker/oryx/commit/f9f51e220cbf4a03ec2111d1789cc8be22b3ff3e))
+* **content:** update content service to possibility use adapters by config ([#755](https://github.com/spryker/oryx/issues/755)) ([43089dc](https://github.com/spryker/oryx/commit/43089dc3326448c58f99978e8c42385f238ccc2f))
+* introduce build time feature level flag, to accomodate breaking changes and feature versioning. ([#725](https://github.com/spryker/oryx/issues/725)) ([d7d6613](https://github.com/spryker/oryx/commit/d7d6613c39554a2e83768815082dbe3bca59a1e9))
+* **presets:** add b2b-storefront preset ([#703](https://github.com/spryker/oryx/issues/703)) ([b62f662](https://github.com/spryker/oryx/commit/b62f6625f5293ec8887a585f901794ca5514f3a9))
+* **site:** breadcrumbs ([#735](https://github.com/spryker/oryx/issues/735)) ([8d9ea37](https://github.com/spryker/oryx/commit/8d9ea37fc1ee4decdd40b8a0357909017a2cfd5b))
```
Comment (tobi-or-not-tobi) [inline libs/CHANGELOG.md]:
```suggestion
* **site:** breadcrumb component ([#735](https://github.com/spryker/oryx/issues/735)) ([8d9ea37](https://github.com/spryker/oryx/commit/8d9ea37fc1ee4decdd40b8a0357909017a2cfd5b))
```

File: libs/CHANGELOG.md [hunk 6]
```diff
@@ -3,6 +3,41 @@
...
+* **search:** facets and browser history ([#770](https://github.com/spryker/oryx/issues/770)) ([b3ba474](https://github.com/spryker/oryx/commit/b3ba474cd44e2df7909e09281010087bcca50a9a))
+* **search:** source for the query that is using for search ([#773](https://github.com/spryker/oryx/issues/773)) ([cd0ddbf](https://github.com/spryker/oryx/commit/cd0ddbfa3ce0dc282100f2c5e6d37adcf947d2f2))
+* **ui:** button states styles ([#771](https://github.com/spryker/oryx/issues/771)) ([7ffd7eb](https://github.com/spryker/oryx/commit/7ffd7ebb417ef59c87cc4e6ee9f4ed642501d96c))
+* **ui:** modal's height in safari ([#738](https://github.com/spryker/oryx/issues/738)) ([4360576](https://github.com/spryker/oryx/commit/4360576d9b24ed53c95b9e26883f4191446d43b3))
+* **ui:** modal's margins and scrolling behaviour  ([#736](https://github.com/spryker/oryx/issues/736)) ([506c84a](https://github.com/spryker/oryx/commit/506c84a41f07bcc56aca72adad9ce9c661e8acd6))
+
+
+### Features
+
+* **content:** resolve product, category and content links ([#775](https://github.com/spryker/oryx/issues/775)) ([f9f51e2](https://github.com/spryker/oryx/commit/f9f51e220cbf4a03ec2111d1789cc8be22b3ff3e))
```
Comment (tobi-or-not-tobi) [inline libs/CHANGELOG.md]:
```suggestion
* **content:** Support product, category and CMS links in link component ([#775](https://github.com/spryker/oryx/issues/775)) ([f9f51e2](https://github.com/spryker/oryx/commit/f9f51e220cbf4a03ec2111d1789cc8be22b3ff3e))
```

File: libs/domain/content/CHANGELOG.md [hunk 7]
```diff
@@ -3,6 +3,23 @@
...
 
+# 1.1.0 (2023-09-22)
+
+
+### Bug Fixes
+
+* blank screen instead of 404 page ([#788](https://github.com/spryker/oryx/issues/788)) ([720eb12](https://github.com/spryker/oryx/commit/720eb12bea26167de0868218f445429ebb3e8e8e))
+
+
+### Features
```
Comment (tobi-or-not-tobi) [inline libs/domain/content/CHANGELOG.md:9]: features first would be great, can you also update the feature commit titles here and all other sub packages?

File: libs/domain/order/CHANGELOG.md [hunk 8]
```diff
@@ -3,6 +3,17 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# 1.1.0 (2023-09-22)
+
+
```
Comment (tobi-or-not-tobi) [inline libs/domain/order/CHANGELOG.md:8]: nitpick; should be one break

File: libs/domain/order/CHANGELOG.md [hunk 9]
```diff
@@ -3,6 +3,17 @@
...
+### Bug Fixes
+
+* clear guest order session storage on login ([#731](https://github.com/spryker/oryx/issues/731)) ([a0bdb81](https://github.com/spryker/oryx/commit/a0bdb810ec81edab111caa21aeac6e14e52ab07d))
+
+
+
+
+
```
Comment (tobi-or-not-tobi) [inline libs/domain/order/CHANGELOG.md:16]: nitpick; should be one break

**Review by tobi-or-not-tobi — CHANGES_REQUESTED** [review]

File: libs/domain/order/CHANGELOG.md (hunk 9 above)
Comment (tolerants) [reply libs/domain/order/CHANGELOG.md:16]: Not really. There are 5 line-breakes between two versions.

File: libs/domain/order/CHANGELOG.md (hunk 8 above)
Comment (tolerants) [reply libs/domain/order/CHANGELOG.md:8]: Not really. There are 2 line-breakes after the version title.

**Review by tobi-or-not-tobi — APPROVED** [review]

**Pull request merged**